packages/platform-sdk-generator/src/generateResource2.ts

```diff
@@ -58,7 +58,9 @@ async function generateMethods(resource: Resource, model: Model) {
   const referencedTypes = new Set<Component>();
 
   for (const method of resource.operations) {
-    const methodName = (method.verb === "delete") ? method.name : method.verb;
+    const methodName = (method.verb === "delete" || method.verb === "load")
```

Comment:
The new loadObject set endpoints for interfaces also use verb load so they conflict in codegen. Used name instead to deconflict

Reply:
Discussing offline